composer.json

```diff
@@ -8,7 +8,7 @@
 		"files": ["lib/Constants.php", "lib/CoreFunctions.php"]
 	},
 	"require": {
-		"thecodingmachine/safe": "^1.0.0",
+		"thecodingmachine/safe": "^1.3.3",
```

Comment:
I don't understand why upgrading Safe is required to get Docker working?

Reply:
It’s not. What we need is a minimum installed package level of safe 1.3.2 to get the site running, due to line https://github.com/standardebooks/web/blob/ebf9651e25b17788a699bc6d09b8613a68b6742d/lib/Constants.php#L4. As it stands, specifying `^1.0.0` will install 1.3.3 due to version resolution, but my usual feeling is that if we need a specific minimum package version we might as well require it.

So what I’m saying is that I’m happy to drop this commit if you’d rather 🙂

Reply:
I just checked and we’re running safe 1.3.3 on the server.

Reply:
OK!